content/docs/oracle/run-oracle/_index.md

```diff
@@ -0,0 +1,190 @@
+- Simulator plugin: for simulated ATN/NTN data. See the `simulator_plugin` adaptor in [`/plugins`<i class='fas fa-external-link-alt'></i>](https://github.com/autonity/autonity-oracle/tree/master/plugins).
+
+### Configure FX data source plugins
+To configure plugins edit the `plugins_conf.yml` file to add a config entry for each plugin.  The oracle server release contains out-the-box multiple plugins for four publicly accessible FX endpoints with free and paid subscriptions tiers. You will need to create an account and get an API Key to connect. One or more plugin source must be configured.
+
+Navigate to the public GitHub repo [autonity-oracle <i class='fas fa-external-link-alt'></i>](https://github.com/autonity/autonity-oracle) `README.md` [Configuration <i class='fas fa-external-link-alt'></i>](https://github.com/autonity/autonity-oracle#configuration) section to view the supported FX endpoint providers.
+
+For each FX endpoint configured:
+
+1. Get FX plugin API Key(s). Navigate to one of the listed FX endpoint websites and create an account. Make a note of the API Key.
+2. Add configuration entry to `plugins-conf.yml`. Navigate to the `config` sub-directory of your installation (default: `./build/bin/config`) and edit the file to add an entry for each plugin you are configuring.
+
+Configuration fields:
+
+| Name | Datatype | Description |
+| :-- | :--: | :-- |
+| `name` | string | the name of the plugin binary; use the name of the sub-directory in the `plugins` directory |
+| `key` | string | the API key granted by your data provider to access their data API |
+| `scheme` | string | the data service http scheme, http or https |
+| `endpoint` | string | the data service endpoint url of the data provider |
+| `timeout` | int | the duration of the timeout period for an API request |
+| `refresh` | int | data update interval. Used for a rate limited provider's plugin to limit the request rate. |
+
+An example minimal entry could be:
+
+```yaml
+- name: forex_currencyfreaks
+  key: 5490e15565e741129788f6100e022ec5
+```
+### Configure ATN/NTN data source plugin
+
+No configuration is required for the testnet ATN/NTN Simulator built when [installing oracle server data source plugins](/oracle/install-oracle/#install-plugin).
+
+### Develop plugins
+Additional data adaptors for any external data source can be developed using the oracle server's plugin template. See:
+
+- Adaptor code template `template_plugin` in [`/plugins`<i class='fas fa-external-link-alt'></i>](https://github.com/autonity/autonity-oracle/tree/master/plugins).
+- Guide for how _To write a new plugin_ using the template in [`/plugins/README`<i class='fas fa-external-link-alt'></i>](https://github.com/clearmatics/autonity-oracle/tree/master/plugins#readme).
+  
+## Run Autonity Oracle Server as Linux daemon service {#run-daemon}
+
+TO DO
+
+## Run Autonity as Docker image {#run-docker}
+
+TO DO
+
```

Comment:
I would suggest tracking TODO items in a Github issue instead of inline source, perhaps a checklist.

When adding todo items in the source, use the phrase `TODO` instead of `TO DO` so that it is easy to grep for and integrated with dev tooling.

Reply:
Agree. Removed in commit 7e7399995f32dfbba5223c56f49a03f567c7a14a.  Issue created to track this here: https://github.com/autonity/docs.autonity.org/issues/88.